### ISSUE TITLE
Include Figure Settings changes in History

### ISSUE THREAD

**Issue by cmkohnen** [Sjoerd1993/Graphs#612]
Currently the change histoy includes only changes to the data itself, howver this fails to track things as changing a style (which changes item properties like color). Also some things like scale should be tracked there as well. Otherwise, the user might be confused as to why changing the scale and hitting undo does undo something completely else.

**Comment by Sjoerd1993** [comment]
I agree here. Historically, the clipboard only contained changes that directly affect the data itself, but since then other actions have been added as well (such as Item properties like changing linestyle). Having the choice of marker in the clipboard, but not the choice of legend or figure style, feels somewhat inconsistent. So indeed Figure settings should be part of the clipboard as well.
